src/rgw/driver/sfs/sqlite/conversion_utils.h

```diff
@@ -124,4 +126,22 @@ void assign_db_value(const SOURCE& source, std::vector<char>& dest) {
   dest = blob_vector;
 }
 
+template <typename COL>
+sqlite_orm::internal::like_t<COL, std::basic_string<char>, const char*>
+prefix_to_like(COL col, const std::string& prefix) {
+  std::string like_expr;
+  like_expr.reserve(prefix.length() + 10);
```

Comment:
Why 10? 
Not important to not approve the PR, but if you want to reserve in advance to avoid future allocations....maybe better to reserve the worst case scenario? (prefix.length() * 2)

10 sounds like a random magic number here. (Or is there a limitation on the length of the prefix? in that case I would name the number)

Reply:
10 is an educated guess on how many substitutions in common object names. Something to change if it ever turns up in profiling that we reallocate a lot here.